package.json

```diff
@@ -15,7 +15,7 @@
     "description": "Genesis Simple Hooks allows you easy access to the 50+ Action Hooks in the Genesis Theme.",
     "author": "StudioPress",
     "authoruri": "http://www.studiopress.com/",
-    "version": "2.3.0",
+    "version": "2.3.1",
```

Comment:
If it's alright, I'm bumping the version here, so we don't need a separate PR for it.

Reply:
Nice! Great idea to prevent a separate, tiny PR 😎